### ISSUE TITLE
Can't get it working

### ISSUE THREAD

**Issue by Harald-v-K** [twrightsman/tiddlywiki-docker#1]
Hi,
I tried your setup:

- Downloading the git repository
(your instruction should also mention that...)
- ran the docker run command
==> the first time, that pulled the container twrightsman/tiddlywiki

Then the following message was given:
```
Digest: sha256:487aa491962fb3b2edd85314c00a2aa98182a69194b21709c6941e40bae28124`
Status: Downloaded newer image for twrightsman/tiddlywiki:latest
d289220431609951b93dabb292d64fc7452484bec0f5920a54b0c4478f03df9c
docker: Error response from daemon: driver failed programming external connectivity on endpoint adoring_hoover (f8b3392d9f33e92404f3499110678ad206cf6e9f6176f7b4710fbf1b63486b90): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:8080:tcp:172.17.0.2:8080: input/output error.

```

In the files, I could not fine a reference to **adoring_hoover** (are they in the Docker-container?)
Running again (a few times) had the same result, but with other
```
C:\project\tiddlywiki-docker>docker run -d -p 8080:8080 -v c:/project/tiddlywiki-docker/pages:/var/tiddlywiki twrightsman/tiddlywiki
96ed21aeaa3e470bf9eb2274427e633a70b82dd2275bff957a7f835c66c073a9
docker: Error response from daemon: driver failed programming external connectivity on endpoint determined_dubinsky (7b74f807d52520be3214ed10a667536f74d8342ae2533379f9c1fdfc6ba9c54a): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:8080:tcp:172.17.0.2:8080: input/output error.
```
here it is **determined_dubinsky**
```
C:\project\tiddlywiki-docker>docker run -d -p 8080:8080 twrightsman/tiddlywiki
a5632c9813256e390b750016ded5c591ce1f51643d4824145664c1b2d788a69b
docker: Error response from daemon: driver failed programming external connectivity on endpoint quirky_visvesvaraya (e3ad50db4f1d00e85eae737bb4ef4d0f5a64cffd90ac65739a5ea2636a0bbdf1): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:8080:tcp:172.17.0.2:8080: input/output error.
```
here it is **quirky_visvesvaraya**
```

C:\project\tiddlywiki-docker>docker run -d -p 8080:8080 twrightsman/tiddlywiki
c168c8593b35bdcc5653572280d473c714b5260f41decfe46706a01428127d81
docker: Error response from daemon: driver failed programming external connectivity on endpoint focused_hodgkin (6470ca7aa00431c56638a45a4aab2a1ef341ea721be8b1e0efb843ae926accee): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:8080:tcp:172.17.0.2:8080: input/output error.
```
here it is **focused_hodgkin**

I expected just the prompt and a web-access to the application in the Docker image...

What am I doing wrong, please explain...

**Comment by twrightsman** [comment]
Hello Harald,
The names that you put in bold are randomly generated docker container names when one isn't provided, they change every time you use `docker run`. You also don't need to download the git repository to use my docker container; it is automatically built and uploaded to the Docker Store, which is where it downloads the container image from when you use `docker run`.

It seems that you are running Docker on Windows. I'm not an expert with Windows, but I believe your path needs to change from `c:/project/tiddlywiki-docker/pages` to something like `C:\\project\tiddlywiki-docker\pages`.

**Comment by Harald-v-K** [comment]
Hi Travis,

Thanks for your explanation. I already thought that they were generated names (they were too funny...).
I indeed tried to make the 'base directory' a windows one and that worked!

I am looking into Docker and Node.JS and thought your 'base' would really start from scratch. Now I see that the Docker container already has the Tiddlywiki 'preinstalled'. That simplifies matters.

I got it working (and now have a bunch of other issues that I need to explore first if I can make it work. The layout of my TW is messed up (somehow the stylesheets don't get involved in building the page...

But thanks again for your help, this issue is closed as far as I'm concerned.
